### ISSUE TITLE
Numbers are returned as word ("one" instead of "1")

### ISSUE THREAD

**Issue by padmalcom** [alphacep/vosk-api#315]
Hi, this is not really an issue but a feature request. At least for German, the numbers are returned as words what makes it very hard to parse automatically. Can we avoid that somehow? I guess only on training data level, right?

**Comment by nshmyrev** [comment]
You have to apply postprocessing which you implemented here: https://github.com/allo-media/text2num/pull/39

**Comment by padmalcom** [comment]
Haha that's what I did as workaround. But I found converting numbers to text is easier than converting text to numbers. That made me ask you if the model could output numerics.
So if there is no way except to change training data, this ticket can be closed.
For German I recommend to use https://github.com/padmalcom/text2numde by the way. This is more elegant and the process to convert words to nubers is transparent.
